maquetado/scss/detail.scss

```diff
@@ -12,19 +14,30 @@ $body-padding-top-for-book: 200px - $body-padding-top-for-back;
 .back {
   display: flex;
   font-size: 20px;
+  margin-bottom: $body-padding-top-for-book - $body-padding-top-for-back;
 
   &:before {
     content: url(./images/back-arrow.png);
     margin-right: 15px;
   }
+
+  @media (max-width: $breakpoint-desktop) {
```

Comment:
instead of deffining a media inside each class we try to define a media query at the bottom of the file with all the classes inside to keep it consistent across all projects
At the bottom of the file:
@media (max-width: $breakpoint-desktop) {
  .back {
    ...
  }
  
  .badge {
    ...
  }
}